frontend_lib/src/helper.js

```diff
@@ -715,9 +715,23 @@ export const checkUsernameValidity = async (apiUrl, username, props) => {
 
 export const formatAbsoluteDate = (rawDate, lang, options = {}) => new Date(rawDate).toLocaleString(lang, options)
 
-// Equality test done as numbers with the following rules:
-// - strings are converted to numbers before comparing
-// - undefined and null are converted to 0 before comparing
+export const readableDateFormat = (rawDate, lang) => {
+  const options = {
+    weekday: 'long', year: 'numeric', month: 'long', day: 'numeric'
+  }
+  const date = new Date(rawDate).toLocaleDateString(lang, options)
+  const time = new Date(rawDate).toLocaleTimeString(lang, { hour: '2-digit', minute: '2-digit' })
```

Comment:
I think we are going to have a problem with Norwegian here. Update your branch with develop and try using formatAbsoluteDate, there are several formats available.

Reply:
Fixed in a40c65b